projects/gnomad/src/client/DownloadsPage.js

```diff
@@ -249,14 +249,33 @@ export default () => (
   <InfoPage>
     <PageHeading>Downloads</PageHeading>
     <p>
-      The variant dataset files below contain all subsets and are large in size. We recommend using{' '}
-      <ExternalLink href={'https://Hail.is/'}>Hail 0.2</ExternalLink> to work with gnomAD data. It
-      is easiest to download these files in parallel on the commandline using{' '}
+      gnomAD is available for download in VCF and Hail Table (.ht) formats. The variant dataset
+      files below contain all subsets (non-neuro, non-cancer, controls only, and non-TOPMed). The
+      files can be downloaded in parallel on the commandline with{' '}
       <ExternalLink href={'https://cloud.google.com/storage/docs/gsutil'}>gsutil</ExternalLink>.
-      After installing gsutil, start navigating with this command:
     </p>
 
-    <code>gsutil ls gs://gnomad-public/release/2.1</code>
+    <p>
+      For example, after installing gsutil, start navigating with this command:{' '}
+      <code>gsutil ls gs://gnomad-public/release/2.1.1</code>
+    </p>
+
+    <p>
+      Download all exome VCF files locally, run:{' '}
+      <code>gsutil -m cp gs://gnomad-public/release/2.1.1/vcf/exomes .</code>
```

Comment:
This command won't work. Since exomes is a directory, `cp` would need a `-r` flag.

Also, I don't think we should encourage users to run this, since this will download both the all sites VCF and the per chromosome VCFs, resulting in ~50 GB of duplicate data.